lib/lighthouse/benefits_claims/service.rb

```diff
@@ -24,6 +25,11 @@ def initialize(icn)
     def get_claims(lighthouse_client_id = nil, lighthouse_rsa_key_path = nil, options = {})
       claims = config.get("#{@icn}/claims", lighthouse_client_id, lighthouse_rsa_key_path, options).body
       claims['data'] = filter_by_status(claims['data'])
+      # https://github.com/department-of-veterans-affairs/va.gov-team/issues/98364
+      # This should be removed when the items are removed by BGS
+      if Flipper.enabled?(:cst_suppress_evidence_requests)
+        claims['data'].each { |claim| suppress_evidence_requests(claim) }
+      end
```

Comment:
Question for @jerekshoe - I see in your [previous comment](https://github.com/department-of-veterans-affairs/vets-api/pull/20368#discussion_r1925610830) that the `get_claims` call should not need this, and rather the call should be made in the `get_claim` method, correct?  I also see that the comment was made on the `get_claim` method line so I wasn't fully sure that one or the other wasn't a typo/mistake. I just want to be sure we're making the update in the right place 😄

Reply:
Good catch, we should only need to worry about suppressing evidence requests in the `get_claim` method as we only get tracked item info when we get the details for an individual claim

Reply:
Ah! Ok, on it.